tests/components/rmvtransport/test_config_flow.py

```diff
@@ -0,0 +1,52 @@
+"""Tests for rmvtransport config flow."""
+from unittest.mock import patch
+
+from homeassistant.components.rmvtransport import config_flow
+from tests.common import mock_coro
+
+from .test_sensor import get_departures_mock, search_station_mock
+
+
+async def test_flow_works(hass, aioclient_mock):
+    """Test that config flow works."""
+    result = await hass.config_entries.flow.async_init(
+        config_flow.DOMAIN, context={"source": "user"}
+    )
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "user"
+
+    with patch(
```

Comment:
Break out these patches into pytest fixture instead, to make the test cleaner and easier to read.